web/js/blog-editor/blog-editor.js

```diff
@@ -83,9 +83,11 @@ function blog_editor_init(name, editor_config) {
 		div_container_editor.empty();
 		div_container_editor.wrap('<div class="blog-content-md-editor"></div>');
 		var blog_contend_md_editor = div_container_editor.parent();
+		blog_contend_md_editor.parent().parent().addClass("card");
 		blog_contend_md_editor.prepend($('<div class="blog-content-md-editor-toolbar"></div>').append(toolbar));
-		div_container_editor.wrap('<div class="blog-content-md-editor-in border"></div>');
-		div_container_editor.append($('<div class="border d-flex justify-content-center align-items-center" style="width: 100%; height: 500px;" />').append('<div class="spinner-border text-muted" style="width: 3rem; height: 3rem;" />'));
+		blog_contend_md_editor.parent().addClass("card");
+		div_container_editor.wrap('<div class="blog-content-md-editor-in"></div>');
+		div_container_editor.append($('<div class="border d-flex justify-content-center align-items-center" style="width: 100%; height: 496px;" />').append('<div class="spinner-border text-muted" style="width: 3rem; height: 3rem;" />'));
 
```

Comment:
Thanks for your contribution, but I'm already working on this, so you don't need to change this.